### ISSUE TITLE
Only black screen and sound, no graphic output

### ISSUE THREAD

**Issue by AngelKing86** [Ryujinx/Ryujinx#2236]
## Bug Report

Only black screen and game sound

### What's the issue you encountered?

The graphics were extremely blended bevore and then I updated the graphics card drivers to AMD Drivers to V21.4.1
I start any game and only get black screen in the main window.

### How can the issue be reproduced?

Start a game like Yoshi or Paper Mario

### Log file

<details>
  <summary>Click to expand!</summary>

00:00:17.074 |I| HLE.OsThread.10 ServiceNv Create: Created map 18 with size 0x00400000!
00:00:17.523 |I| HLE.OsThread.10 ServiceNv Create: Created map 19 with size 0x02000000!
00:00:18.093 |I| HLE.OsThread.8 AudioRenderer AcquireSessionId: Registered new output (0)
00:00:18.172 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:18.172 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:18.174 |I| HLE.OsThread.8 AudioRenderer ReleaseSessionId: Unregistered output (0)
00:00:18.174 |I| HLE.OsThread.8 AudioRenderer AcquireSessionId: Registered new output (0)
00:00:18.179 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:18.199 |S| HLE.OsThread.5 ServiceAm SetScreenShotPermission: Stubbed. {screenShotPermission: True}
00:00:18.668 |I| HLE.OsThread.10 ServiceNv Create: Created map 20 with size 0x00400000!
00:00:18.670 |I| HLE.OsThread.10 ServiceNv Create: Created map 21 with size 0x00400000!
...
00:00:19.144 |I| HLE.OsThread.10 ServiceNv Create: Created map 30 with size 0x00800000!
00:00:19.144 |I| HLE.OsThread.10 ServiceNv Create: Created map 31 with size 0x00400000!
00:00:19.145 |I| HLE.OsThread.10 ServiceNv Create: Created map 32 with size 0x00400000!
00:00:19.186 |I| HLE.OsThread.10 ServiceNv Create: Created map 33 with size 0x00400000!
00:00:19.232 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.233 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.236 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.237 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.237 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.237 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.239 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
...
00:00:19.250 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.250 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.253 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.253 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.256 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.256 |W| HLE.GuestThread.16 KernelSvc PrintResult: SendSyncRequest64 returned error PortRemoteClosed.
00:00:19.481 |I| HLE.OsThread.10 ServiceNv Create: Created map 34 with size 0x01000000!
00:00:19.481 |I| HLE.OsThread.10 ServiceNv Create: Created map 35 with size 0x00800000!
00:00:19.482 |I| HLE.OsThread.10 ServiceNv Create: Created map 36 with size 0x01000000!
00:00:40.425 |I| HLE.OsThread.10 ServiceNv Create: Created map 121 with size 0x00400000!
00:00:40.435 |I| HLE.OsThread.10 ServiceNv Create: Created map 122 with size 0x00040000!
00:00:40.441 |I| HLE.OsThread.10 ServiceNv Create: Created map 123 with size 0x00400000!
00:00:40.442 |I| HLE.OsThread.10 ServiceNv Create: Created map 124 with size 0x00400000!
00:00:40.443 |I| HLE.OsThread.10 ServiceNv Create: Created map 125 with size 0x00400000!
00:00:40.582 |I| HLE.OsThread.10 ServiceNv Create: Created map 126 with size 0x00400000!
00:00:40.587 |I| HLE.OsThread.10 ServiceNv Create: Created map 127 with size 0x00040000!

00:00:40.798 |W| HLE.OsThread.10 ServiceNv Wait: GPU processing thread is too slow, waiting on CPU...

00:00:41.797 |E| HLE.OsThread.10 Gpu WaitOnSyncpoint: Wait on syncpoint 1 for threshold 4742 took more than 1000ms, resuming execution...
00:00:42.505 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 42!
00:00:42.506 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 41!
00:00:42.506 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 32!
00:00:42.506 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 31!
00:00:42.507 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 30!
00:00:42.507 |I| HLE.OsThread.10 ServiceNv DecrementMapRefCount: Deleted map 29!
</details>
### Environment?

- Ryujinx version: 1.0.6845
- Game version: 1.0.1
- System Specs:
  - OS: Windows 10
  - CPU: AMD Ryzen 5 3600
  - GPU: AMD Radeon RX 5700 XT
  - RAM: 16GB
- Applied Mods : nothing

**Comment by AcK77** [comment]
It could be nice to know which game are you trying to run?

**Comment by AngelKing86** [comment]
I have tested with Yoshis Crafted World, Papier Mario Origami King and with Zelda. I can hear sound and see a FPS but the window is only black

**Comment by EmulationFanatic** [comment]
The latest AMD drivers break all graphics in Ryujinx. It is recommended to roll back to the previous driver release.

**Comment by Thecoolsun** [comment]
Try putting the games in fullscreen mode it worked for me

**Comment by jochuan** [comment]
Tried the above on Rune Factory 4 and still black screen but i was able to get sound output.

**Comment by AngelKing86** [comment]
I can confirm, the problem still exists also in build 1.0.6875. No graphical output. Only sound.

**Comment by Quackdoc** [comment]
I am also getting this on both Rune factory 4, and story of seasons on build 1.0.6878 with AMD driver 21.4.1

**Comment by AcK77** [comment]
As @EmulationFanatic said, there is currently an issue with Ryujinx on latest AMD drivers, as a temporary fixes you can rollback to drivers 21.3.1 and games should render something.

**Comment by HomeStarRunnerTron** [comment]
Just wanted to leave a message that as a Bootcamp user on MacBook Pro using BootCampDrivers.com, indeed, as of 5/27/2021, the newest AMD driver up causes a black screen on Ryujinx, but going back to their 21.3.1 driver worked perfectly.[ That one's over here, in-case this is pertinent information for anybody.](https://bootcampdrivers.freeforums.net/thread/931/adrenalin-march-2021-21-available)

**Comment by MHAdnan** [comment]
I Tried pokemon cafe mix but only the mint colour(emulator theme) is appearing.if i click somewhere I can hear sound but I cant see anything.Game fps is 100 plus. Pokemon Unite has the same problem but this time, its a black screen and no fps.100.00 percent FIFO. Whats wrong with me!!

**Comment by CabTheProgrammer** [comment]
I am having this issue but I have integrated graphics. I am trying to run on a laptop with a 5th gen i3 processor

**Comment by AmidoNewsInside** [comment]
same issue, but with nvidia latest card rtx 3050

**Comment by edisionnano** [comment]
> same issue, but with nvidia latest card rtx 3050

reset your nvidia settings

**Comment by Wfree86** [comment]
This still happens to me with Super Smash Bros Ultimate. 

 i7 4700
Nvidia GTX 850m
16gb RAM

**Comment by Wfree86** [comment]
@Thog since you're the one to mark it complete, and Vulkan has now merged I thought I would try to get eyes on this to see if this could be reopened and Vulkan tagged.

**Comment by Thog** [comment]
> @Thog since you're the one to mark it complete, and Vulkan has now merged I thought I would try to get eyes on this to see if this could be reopened and Vulkan tagged.

This is a separated issue here.

The bug that was initially reported here was for a graphic bug specific to OpenGL and AMD drivers.

Please open a new issue
